### ISSUE TITLE
Enhance Brave top site icons - in centre

### ISSUE THREAD

**Issue by mlntr** [brave/brave-browser#30668]
Website icons are typically the most useful/prominent part of a new tab page. Brave's are less good (mainly for desktop)

* Put icon in centre (could be an option)
  * Coming from other browsers, it's weird to see it in the corner
  * wasting space
<br>

* also allow users up to 4 rows for favourites icons
  * 12 isn't many
  * tab pages are annoying
  * wasting vertical space
  * in narrow screen, those extra icons could then go to a tab page
<br>

![Brave yes no](https://github.com/brave/brave-browser/assets/17552475/d2ba41d6-f752-49b2-bf89-c5cd09046c08)

**Comment by metablaster** [comment]
Absolutely yes.

It's incorrect to assume that users visit only 5 sites.

**Comment by SD191100** [comment]
can anyone help me with starting contributions on brave... i use brave all the time and as a web dev I would like to help it make better..

**Comment by mmseng** [comment]
Yes, please. This is currently my single biggest hangup with switching to Brave. Pagination is an awful solution and I don't see a reason to arbitrarily limit the number of bookmarks/top sites in the first place. Worst case it should scroll to show anything that can't fit in whatever area you define for it.

**Comment by dpisati** [comment]
Looking forward to this feature...

![image](https://github.com/brave/brave-browser/assets/43629737/4ada507f-075a-4be8-843c-b76297320663)
☝🏼 😠

**Comment by henrikvilhelmberglund** [comment]
I think every other browser does this better than Brave.

**Comment by cssyphus** [comment]
YES - allow more favorite sites
YES - pagination is a poor solution - why limit? Use the screen space if the user desires!
MEH - Positioning of the Favorite Sites does NOT matter to me (top left is fine... page center is fine...)

Sure, there are other bookmark locations (bookmark bar). But when I click a New Tab and am looking at the New Tab Page, I would love to have 20 - 30 Favorited sites that are immediately clickable. 

This is a big, every-day, annoyance to me - but it still took me 3 years to post this note...

**Comment by geraldsnm** [comment]
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION 
YES +++++++++++++++++++++++++++++++++++++++++++1 MILLION

**Comment by geraldsnm** [comment]
At least give us a manual config option even if it's buried in the config somewhere to make this customizable

**Comment by MisterT87** [comment]
+1 also looking for this feature

**Comment by bsclifton** [comment]
cc: @aguscruiz

**Comment by IPPYZ** [comment]
cc: @aguscruiz @rebron do it

**Comment by Schanzation** [comment]
I like the way it looks on my desktop. However. I hate clicking those small dots to switch between pages. Please make it scrollable with mousewheel.

**Comment by ayhanavci** [comment]
Greetings from November 2024. We still need more top site icons instead of the tiny dots.

**Comment by Prototype-Integral** [comment]
Still waiting for this

**Comment by johnkanderson** [comment]
> Still waiting for this

Same...

**Comment by c-h-b** [comment]
This is really annoying! Please allow more top sites icons.

**Comment by ayhanavci** [comment]
This looks much better now that there are 24 icons on center-top. Update to latest Brave and enable " New Tab Page refresh" on brave://flags/ Feature added on v1.83.108, I'm on v1.83.118 as of this post.
Tabs are still not scrollable with mouse and you have to click the dot to go to 2nd page and bigger icons would be better. But good progress nonetheless.

**Comment by bsclifton** [comment]
Closing as we've implemented! Thanks 😄

**Comment by escalonn** [comment]
"Frequently visited" appears to still be limited to 12 sites. This is unsatisfactory for all the same reasons & I consider this issue still open.
